outputs.tf

```diff
@@ -1,13 +1,29 @@
+output "alb_security_group" {
+  description = "Security Group object assigned to the ALB."
+  value       = aws_security_group.alb_sg
+}
+
 output "alb_security_group_id" {
   description = "Security Group ID assigned to the ALB."
   value       = aws_security_group.alb_sg.id
 }
 
+output "alb_target_group" {
+  description = "Object of the first target group with HTTPS listener"
+  value       = aws_lb_target_group.https
+}
+
 output "alb_target_group_id" {
   description = "ID of the target group with the HTTPS listener."
   value       = aws_lb_target_group.https.id
 }
 
+
+output "alb_target_group_2" {
+  description = "Object of the second target group with HTTPS listener"
+  value       = aws_lb_target_group.https2
+}
+
```

Comment:
I notice we output the target group id for the main target group, do we want to export the id for the second as well?

Reply:
Good question, I was mainly keeping the `alb_target_group_id` to be backwards compatible with existing deployments. I don't mind throwing that in there for the second group but as a practice should we be outputting objects or primitives such as strings etc?